depthai_nodes/parser_generator.py

```diff
@@ -52,8 +53,13 @@ def build(self, nn_archive: dai.NNArchive, head_index: int = None) -> Dict:
                     f"Parser {parser_name} does not inherit from BaseParser class."
                 )
 
-            head = decode_head(head)
-            parsers[index] = pipeline.create(parser).build(head)
+            head_config = decode_head(head)
+            head_config["model"] = {"inputs": []}
```

Comment:
I would not do a nested dictionary just for inputs. The entire point of decode_head is to get a dictionary of key-value pairs that has zero nestings. Rather just add `head_config["inputs"] = []` and then do the rest.

Reply:
Added in [c781241](https://github.com/luxonis/depthai-nodes/pull/126/commits/c781241884b3774880f93e11dc75d0f01dd48478).